### ISSUE TITLE
Distributed Minio: 1 of 4 nodes hogging cpu and ram

### ISSUE THREAD

**Issue by gitwittidbit** [minio/minio#12768]
I have set up distributed minio and all four nodes are up and running. There is no in/out traffic yet. Three nodes are using very little ram and almost no cpu. One node is using a lot ram and a lot of cpu cycles.

## Expected Behavior

I would expect to see more or less no cpu and ram usage on all four nodes (due to no traffic)

## Current Behavior

One node is using a lot ram and a lot of cpu cycles.

## Steps to Reproduce (for bugs)

1. Set up distributed minio
2. start distributed minio
3.
4.

## Regression

 don't know. This is the first working installation I managed to create.

## Your Environment

* Version used (`minio --version`):
* Server setup and configuration:
* Operating System and version (`uname -a`):
Minio is the latest (as of three days ago)
Running in lxc on PVE 6.4.9 (each node in a separate lxc) - this is still the test setup

**Comment by harshavardhana** [comment]
> I have set up distributed minio and all four nodes are up and running. There is no in/out traffic yet. Three nodes are using very little ram and almost no cpu. One node is using a lot ram and a lot of cpu cycles.

Really you haven't shared anything here for us to help you can you provide `mc admin profile` etc? 

> * Version used (`minio --version`):
> * Server setup and configuration:
> * Operating System and version (`uname -a`):
>   Minio is the latest (as of three days ago)
>   Running in lxc on PVE 6.4.9 (each node in a separate lxc) - this is still the test setup

what your disk here?

**Comment by gitwittidbit** [comment]
>Really you haven't shared anything here for us to help you can you provide mc admin profile etc?

Attached
[profile.zip](https://github.com/minio/minio/files/6857964/profile.zip)

>what your disk here?

Each node has a separate partition on a local drive mounted into its lxc

**Comment by harshavardhana** [comment]
Looks like its the embedded console that is using up all the memory here 

```
go tool pprof -alloc_space profile-192.168.99.53:9000-mem.pprof
File: minio
Type: alloc_space
Time: Jul 21, 2021 at 11:04am (PDT)
Entering interactive mode (type "help" for commands, "o" for options)
(pprof) top
Showing nodes accounting for 6495700594.33kB, 99.74% of 6512772144.38kB total
Dropped 2593 nodes (cum <= 32563860.72kB)
      flat  flat%   sum%        cum   cum%
6495700590.24kB 99.74% 99.74% 6495700611.48kB 99.74%  github.com/minio/console/restapi.adminClient.serverHealthInfo
    4.09kB 6.3e-08% 99.74% 6495700615.57kB 99.74%  github.com/minio/console/restapi.startHealthInfo
         0     0% 99.74% 6495700615.57kB 99.74%  github.com/minio/console/restapi.(*wsAdminClient).healthInfo
```

Have you run something from the console? @gitwittidbit

**Comment by gitwittidbit** [comment]
>Have you run something from the console? @gitwittidbit

Only thing that comes to mind is the diagnostic (out of curiosity - at that time there were no issues I was aware of). 

This took ages to complete (and I am not even sure it did complete - as no info was shown - the only change was that the spinning circle was gone after I looked again after a while).

**Comment by harshavardhana** [comment]
> This took ages to complete (and I am not even sure it did complete - as no info was shown - the only change was that the spinning circle was gone after I looked again after a while).

Yes that has caused the memory usage sent a fix here https://github.com/minio/console/pull/887

**Comment by harshavardhana** [comment]
This is fixed here https://github.com/minio/console/pull/887 - will be available in the next release.

**Comment by gitwittidbit** [comment]
>This is fixed here minio/console#887 - will be available in the next release.

Wow, that was quick - thanks!

Until the next release comes out - is it sufficient to just restart the offending node and not run diagnostics anymore?

**Comment by harshavardhana** [comment]
> Until the next release comes out - is it sufficient to just restart the offending node and not run diagnostics anymore?

yes that is enough.
